ci/scripts/ci-deploy-staging.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# This script is used to deploy collector on demo account cluster
+
+set -euo pipefail
+IFS=$'\n\t'
+set -x
+
+install_collector() {
+	# Set the namespace and release name
+	release_name="opentelemetry-collector"
+	namespace="otel-staging"
```

Comment:
This will work for https://github.com/DataDog/opentelemetry-demo/blob/prod/ci/values.yaml#L50, but not here: https://github.com/DataDog/opentelemetry-collector-contrib/blob/prod/ci/values.yaml#L116. 

We need to have env  in dd exporter pull dynamically from:
```
    - name: OTEL_K8S_NAMESPACE
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.namespace
```
as well.

Reply:
@dineshg13 heads up I pushed a commit that does what is mentioned here. The hosts are now tagged with this: https://app.datadoghq.com/infrastructure?tags=env%3Aotel-staging, and it also looks like the duplicate env tags go away on their own: [link](https://app.datadoghq.com/metric/explorer?graph_layout=multi&start=1692190078829&end=1692190667000&paused=true#N4Ig7glgJg5gpgFxALlAGwIYE8D2BXJVEADxQEYAaELcqyKBAC1pEbghkcLIF8qo4AMwgA7CAgg4RKUAiwAHOChASAtnADOcAE4RNIKtrgBHPJoQaUAbVBGN8qVoD6gnNtUZCKiOq279VKY6epbINiAiGOrKQdpYZAYgUJ4YThr42gDGSsgg6gi6mZaBZnHKGABuMMg4CHBoAHTJCBhQODBOcMQO2nXaDfmFGg3aeCJiIjDAjDgaCMgQALQADADMAGwAHK1Qm3CrAEybmUeCAOw8AAQARliXwHAiFTwgPAC6VK7ueJih4V+qH4YGKleKvD4gOZYNA5UDyDAwhB1ZRtDqZX4aCCZRJoURwJxyRTKdK4qA4vFOehMZQiNweNCvfgQeyYLAEhQ5EDXDDaSzvHh8SHyXEIADCUmEMBQIh+aB4QA).